test/test_jit.py

```diff
@@ -13412,6 +13412,25 @@ def foo(a):
 
         FileCheck().check("aten::rand").check("aten::add").run(str(foo.graph))
 
+    def test_mobile_code(self):
+        @torch.jit.script
+        def foo():
+            return torch.rand(2, 3)
+
+        mobileCode = torch._C.MobileCode(foo.graph, "foo")
+        instructions = mobileCode.instructions()
+        expected = [('LOADC', 1, 0),
+                    ('LOADC', 2, 0),
+                    ('LIST_CONSTRUCT', 0, 2),
```

Comment:
@cccclai  do you think mobile will need to manually add offset to this number?

Reply:
Chatted offline. Let's also have type table. Maybe constant is needed, depending how bytecode looks like

Reply:
sounds good.

Reply:
Sharing an example bytecode table, we can probably use the same format
```
  (('instructions',
    (('STOREN', 1, 2),
     ('DROPR', 1, 0),
     ('LOAD', 2, 0),
     ('LOADC', 2, 0),
     ('LOADC', 1, 0),
     ('LIST_CONSTRUCT', 0, 1),
     ('OP', 0, 0),
     ('LOADC', 1, 0),
     ('LIST_CONSTRUCT', 1, 1),
     ('OP', 1, 0),
     ('TUPLE_CONSTRUCT', 2, 0),
     ('DICT_CONSTRUCT', 2, 2),
     ('LOAD', 2, 0),
     ('LOADC', 3, 0),
     ('OP', 2, 0),
     ('OP', 3, 0),
     ('LOADC', 3, 0),
     ('OP', 4, 0),
     ('LIST_CONSTRUCT', 3, 1),
     ('OP', 5, 0),
     ('LOAD', 2, 0),
     ('LOADC', 3, 0),
     ('OP', 6, 0),
     ('OP', 7, 0),
     ('LOADC', 3, 0),
     ('OP', 8, 0),
     ('LIST_CONSTRUCT', 4, 1),
     ('OP', 9, 0),
     ('TUPLE_CONSTRUCT', 2, 0),
     ('LOAD', 2, 0),
     ('LOADC', 3, 0),
     ('OP', 10, 0),
     ('OP', 11, 0),
     ('LOADC', 3, 0),
     ('OP', 12, 0),
     ('LIST_CONSTRUCT', 5, 1),
     ('OP', 13, 0),
     ('LOAD', 2, 0),
     ('LOADC', 3, 0),
     ('OP', 14, 0),
     ('OP', 15, 0),
     ('LOADC', 3, 0),
     ('OP', 16, 0),
     ('LIST_CONSTRUCT', 6, 1),
     ('OP', 17, 0),
     ('TUPLE_CONSTRUCT', 2, 0),
     ('LOAD', 2, 0),
     ('LOADC', 3, 0),
     ('OP', 18, 0),
     ('OP', 19, 0),
     ('LOADC', 3, 0),
     ('OP', 20, 0),
     ('LIST_CONSTRUCT', 7, 1),
     ('OP', 21, 0),
     ('MOVE', 2, 0),
     ('LOADC', 3, 0),
     ('OP', 22, 0),
     ('OP', 23, 0),
     ('LOADC', 3, 0),
     ('OP', 24, 0),
     ('LIST_CONSTRUCT', 8, 1),
     ('OP', 25, 0),
     ('TUPLE_CONSTRUCT', 2, 0),
     ('LOADC', 2, 0),
     ('LOADC', 1, 0),
     ('LIST_CONSTRUCT', 9, 1),
     ('OP', 26, 0),
     ('LOADC', 1, 0),
     ('LIST_CONSTRUCT', 10, 1),
     ('OP', 27, 0),
     ('LOADC', 1, 0),
     ('LIST_CONSTRUCT', 11, 1),
     ('OP', 28, 0),
     ('TUPLE_CONSTRUCT', 3, 0),
     ('DICT_CONSTRUCT', 12, 2),
     ('NAMED_TUPLE_CONSTRUCT', 13, 6),
     ('RET', 0, 0))),
   ('operators',
    (('aten::zeros', '', 1),
     ('aten::zeros', '', 1),
     ('prim::TupleIndex', '', 2),
     ('aten::size', '', 1),
     ('aten::__getitem__', 't', 2),
     ('aten::ones', '', 1),
     ('prim::TupleIndex', '', 2),
     ('aten::size', '', 1),
     ('aten::__getitem__', 't', 2),
     ('aten::ones', '', 1),
     ('prim::TupleIndex', '', 2),
     ('aten::size', '', 1),
     ('aten::__getitem__', 't', 2),
     ('aten::ones', '', 1),
     ('prim::TupleIndex', '', 2),
     ('aten::size', '', 1),
     ('aten::__getitem__', 't', 2),
     ('aten::ones', '', 1),
     ('prim::TupleIndex', '', 2),
     ('aten::size', '', 1),
     ('aten::__getitem__', 't', 2),
     ('aten::zeros', '', 1),
     ('prim::TupleIndex', '', 2),
     ('aten::size', '', 1),
     ('aten::__getitem__', 't', 2),
     ('aten::ones', '', 1),
     ('aten::zeros', '', 1),
     ('aten::zeros', '', 1),
     ('aten::zeros', '', 1))),
   ('constants', (None, 1, -1, 0)),
   ('types',
    ('List[int]',
     'List[int]',
     'Dict[int, Tuple[Tensor, Tensor]]',
     'List[int]',
     'List[int]',
     'List[int]',
     'List[int]',
     'List[int]',
     'List[int]',
     'List[int]',
     'List[int]',
     'List[int]',
     'Dict[int, Tuple[Tensor, Tensor, Tensor]]',)),
   ('register_size', 2)),
```

Reply:
ok